### ISSUE TITLE
[XML] Signature is broken

### ISSUE THREAD

**Issue by pfirpfel** [camptocamp/pyramid_oereb#596]
pyramid_oereb/lib/renderer/extract/templates/xml/extract.xml:

```
%if extract.electronic_signature:
    <data:Signiture>${extract.electronic_signature}</data:Signiture>
%endif
```

Beyond the obvious typo, this type is way more complex than just a simple string, see: http://www.w3.org/TR/xmldsig-core/xmldsig-core-schema.xsd

**Scope update:** goal of this issue is simply to fix the syntax of the XML signature.

**Comment by vvmruder** [comment]
Thats true. But... I would give this low priority. Because signature is not well defined at all by federal documents. First we need some information how this should be technically solved.

@voisardf: What do you think?

**Comment by voisardf** [comment]
For us and JU I agree, it's not the highest priority. I don't know about the other pyramid_oereb partners? What about SZ @ralphschulerSZ and ZG @andreasneumann ?

I think that at least GE (and maybe ZH) implemented this feature. What about BE @peterschaer?

**Comment by jwkaltz** [comment]
Fixed (typo part) in #738
